authenticator/authenticator.go

```diff
@@ -34,5 +42,24 @@ func NewAuthenticator(readKeyID, readKeySecret, writeKeyID, writeKeySecret strin
 			[]string{"(request-target)", "nonce", "digest"},
 		),
 	)
-	return auth
+	return auth, nil
+}
+
+// ParseKeyPair parse keyPair to accessKeyID and secretAccessKey
+func ParseKeyPair(keyPair KeyPair) (accessKeyID string, secretAccessKey string, err error) {
```

Comment:
this look unnecessary. Maybe we could make `KeyPair` a struct with 2 fiels "AccessKeyID", "SecretAccessKey"?

Reply:
I agree that we should create struct `KeyPair` and this function will return `(KeyPair, error)`, I think this function is necessary, cuz I see in other project we still have to validate the AccessKeyID and SecretAccessKey if they are empty. So we can use this common function instead of repeat validate code in every project need use httpsign-utils

Reply:
We don't need to parse at all, just make `NewAuthenticator` accepts `KeyPair` directly.

Reply:
fixed in https://github.com/KyberNetwork/httpsign-utils/pull/2/commits/284ce465dc838f84a5f26c54bb9c4fb6e93daed7